torch/quantization/fx/quantization_patterns.py

```diff
@@ -449,7 +451,15 @@ def convert(self, quantizer: QuantizerCls, node: Node, load_arg: Callable,
                 "dtype combination: {} is not "
                 "supported by Linear "
                 "supported dtype combinations are: {}".format(dtypes, supported_dtypes))
-            return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
+            if self.relu_node:
+                op_out = quantizer.quantized_graph.node_copy(self.linear_node, load_arg(quantized=False))
+                relu_args = [op_out]
+                relu_args.extend(load_arg(quantized=False)(self.relu_node.args[1:]))
+                relu_kwargs = load_arg(quantized=False)(self.relu_node.kwargs)
+                return quantizer.quantized_graph.create_node(
+                    "call_function", torch.nn.functional.relu, tuple(relu_args), relu_kwargs)
+            else:
+                return quantizer.quantized_graph.node_copy(node, load_arg(quantized=None))
```

Comment:
just for my own understanding, this is an improvement which is not related to the title of this PR?  Sounds good, just making sure i'm understanding things correctly

Reply:
yeah, this is unrelated to the PR, more of supporting the case when an unsupported qconfig is configured for linear and make sure we still produce correct model and not error out.